CHANGELOG.md

```diff
@@ -24,6 +24,7 @@ For the 2.x changelog see the [viur/server](https://github.com/viur-framework/se
 - killSessionByUser function
 
 ### Removed
+- conf['viur.debug.traceQueries'] flag. It has to be set on viur-datastore directly
```

Comment:
Can you briefly describe how to set the flag there from a viur application?

Reply:
Put in your `main.py`
```py
from viur.core import db
db.config["traceQueries"] = True
```

This worked fine for me.